docs/source/joblib.rst

```diff
@@ -9,14 +9,14 @@ Joblib_ is a library for simple parallel programming primarily developed and
 used by the Scikit Learn community.  As of version 0.10.0 it contains a plugin
 mechanism to allow Joblib code to use other parallel frameworks to execute
 computations.  The ``dask.distributed`` scheduler implements such a plugin in
-the ``distributed.joblib`` module and registers it appropriately with Joblib
+the ``dask_ml.joblib`` module and registers it appropriately with Joblib
 when imported.  As a result, any joblib code (including many scikit-learn
 algorithms) will run on the distributed scheduler if you enclose it in a
 context manager as follows:
 
 .. code-block:: python
 
-   import distributed.joblib
+   import dask_ml.joblib  # import registers joblib plugin
```

Comment:
Could we change the comment to `# joblib plugin registered on import`. Seeing `import ...` threw me off for a second since it looked like the start of another module import statement.

Reply:
Changed it to `registers joblib plugin`